docs/source/amp.rst

```diff
@@ -0,0 +1,28 @@
+.. role:: hidden
+    :class: hidden-section
+
+Automatic Mixed Precision package - torch.cuda.amp
+==================================================
+
+.. automodule:: torch.cuda.amp
+.. currentmodule:: torch.cuda.amp
+
+``torch.cuda.amp`` provides convenience methods for mixed precision.  Mixed precision uses ``torch.float16``
```

Comment:
... methods for running networks with 'mixed' precision, where some operations use the float32 or 'float' datatype and other operations use float16 or 'half.' Some operations, like linear layers and convolutions, are much faster in half precision. Other operations, like reductions, often require the dynamic range of float precision. Networks running in mixed precision try to match each operation to its appropriate datatype.

(Pls kill the NVIDIA-specific reference.)